addons/mail/static/src/components/link_preview_card_view/link_preview_card_view.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-name="mail.LinkPreviewCardView" owl="1">
+        <div class="o_LinkPreviewCardView card position-relative w-100 mb-2 rounded border border-light bg-white shadow-sm overflow-hidden" t-att-class="{ 'me-2': !linkPreviewCardView.linkPreviewListViewOwner.messageViewOwner.isInChatWindow }" t-attf-class="{{ className }}" t-on-mouseenter="linkPreviewCardView.onMouseEnter" t-on-mouseleave="linkPreviewCardView.onMouseLeave" t-ref="root">
+            <div class="row g-0 flex-row-reverse h-100">
+                <div class="col min-w-0" t-att-class="{ 'd-flex align-items-center': !linkPreviewCardView.linkPreview.og_description }">
+                    <div class="o_LinkPreviewCardView_content p-3 bg-white">
+                        <h6 class="o_LinkPreviewCardView_title card-title mb-0 me-2" t-attf-class="{{ linkPreviewCardView.linkPreview.og_description ? 'text-truncate' : 'o_LinkPreviewCardView_hasMultineLines overflow-hidden' }}">
+                            <a t-att-href="linkPreviewCardView.linkPreview.source_url" target="_blank" t-esc="linkPreviewCardView.linkPreview.og_title"/>
```

Comment:
I believe `t-out` is (now) supported by owl. Don't use `t-esc`, it's on its way out.

Reply:
Did I missed the email that deprecate `t-esc` ?